lib/Configuration.php

```diff
@@ -107,7 +107,9 @@ public static function getVersion() {
 
 		$headFile = '.git/HEAD';
 
-		if(file_exists($headFile)) {
+		$is_readable = @is_readable($headFile);
```

Comment:
The variable `$is_readable` is not necessary (see comment below).
Please add a short comment to why `@` is needed here.

i.e. https://github.com/RSS-Bridge/rss-bridge/blob/1cfe939927ca2f62c063ba55b656f7211a5d9c6d/lib/contents.php#L196